applications/ExaquteSandboxApplication/tests/test_ExaquteSandboxApplication.py

```diff
@@ -42,7 +48,7 @@ def AssembleTestSuites():
     allSuite = suites['all']
     allSuite.addTests(
         KratosUnittest.TestLoader().loadTestsFromTestCases([
-            KratosExaquteSandboxGeneralTests
+            KratosExaquteSandboxGeneralTests,TimeAveragingProcessTests
         ])
     )
```

Comment:
Do you need these lines? Are these test the same that have been added in the small suite?

Reply:
If I do not add this, the test `TimeAveragingProcessTests` does not run

Reply:
Try adding these lines instead, it should work I think:
```

    allSuite = suites['all']
    allSuite.addTests(nightSuite)
```

Reply:
Done, it works